src/pudl_archiver/archivers/phmsagas.py

```diff
@@ -0,0 +1,51 @@
+"""Download PHMSHA data."""
+import re
+import typing
+from datetime import date
+from pathlib import Path
+
+from pudl_archiver.archivers.classes import (
+    AbstractDatasetArchiver,
+    ArchiveAwaitable,
+    ResourceInfo,
+)
+
+BASE_URL = "https://www.phmsa.dot.gov/data-and-statistics/pipeline/gas-distribution-gas-gathering-gas-transmission-hazardous-liquids"
+current_year = date.today().year
```

Comment:
Is current year used anywhere?

Reply:
This was an artifact of the end_year partition, good catch. Deleted.